src/variants/lambdatreasury/implementation.mligo

```diff
@@ -0,0 +1,78 @@
+// SPDX-FileCopyrightText: 2022 Tezos Commons
+// SPDX-License-Identifier: LicenseRef-MIT-TC
+
+#if !VARIANT
+#define VARIANT
+
+#include "error_codes.mligo"
+#include "helper/unpack.mligo"
+#include "variants/lambda/types.mligo"
+#include "variants/lambda/common.mligo"
+#include "types.mligo"
+
+// Incorrect token amounts locked
+let wrong_token_amount_err_msg = "WRONG_TOKEN_AMOUNT"
+
+// Proposal size is bigger than `max_proposal_size`
+let large_proposal_err_msg = "LARGE_PROPOSAL"
```

Comment:
Likewise, both here and in `lambdaregistry`, why not import these from `src/common/errors.mligo`?

Reply:
Done